packages/graphback-cli/src/components/config.ts

```diff
@@ -13,11 +13,12 @@ export async function installDependencies(database: string): Promise<void> {
   await execa('npm', ['i'])
   if (database === 'pg') {
     await execa('npm', ['i', '-S', 'pg'])
-  } else if (database === 'sqlite3') {
-    await execa('npm', ['i', '-S', 'sqlite3'])
+    await execa('npm', ['i', '-S', '@graphback/runtime-knex'])
+  } else if (database === 'MongoDB') {
+    await execa('npm', ['i', '-S', 'mongodb'])
```

Comment:
WOW WE FOUND BUG. Both knex and mongo runtime packages have now Pg and mongo. They do not need them and we can have them as peer dependencies

Reply:
Ok. Will add mongodb and pg as peer dependencies in mongo and knex runtime packages respectively.